Makefile

```diff
@@ -124,9 +124,12 @@ unit_test_others: tools_setup
 
 check: tools_setup fmt lint terror_check tidy_mod
 
-fmt:
+fmt: tools_setup
 	@echo "gofumports"
 	tools/bin/gofumports -w -d -local $(PACKAGE_NAME) $(PACKAGE_DIRECTORIES) 2>&1 | awk '{print} END{if(NR>0) {exit 1}}'
+	@echo "shfmt"
+	tools/bin/shfmt  -l -w tests/
```

Comment:
```suggestion
	tools/bin/shfmt -l -w tests/
```

Reply:
should we failed the `fmt` since it's called in `check` and `check` means not silently exit 🤔

maybe `shfmt -d`

Reply:
i change the order of stage `check` in makefile

i think it's better to run `lint` before `fmt`, bcause `lint` will failed when golangci/shfmt find issue in our code.